src/pages/spawn/spawnHost/SpawnHostTableActions.tsx

```diff
@@ -24,18 +24,27 @@ const CopySSHCommandButton: React.FC<{ host: MyHost }> = ({ host }) => {
   const spawnAnalytics = useSpawnAnalytics();
 
   return (
-    <Tooltip placement="top" title="Copied!" trigger="click">
-      <PaddedButton // @ts-expect-error
-        onClick={(e: React.MouseEvent) => {
-          e.stopPropagation();
-          copyToClipboard(sshCommand);
-          spawnAnalytics.sendEvent({ name: "Copy SSH Command" });
-        }}
-        size={Size.XSmall}
+    <FlexContainer onClick={(e: React.MouseEvent) => e.stopPropagation()}>
```

Comment:
Does this need to be a flex container?

Reply:
I also had a question about this because if I put it in a normal div it gives a couple of errors. One of the errors was that it needed a role??

Reply:
Thats for accessibility, https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role 
In this case we dont need it because we aren't using it as a clickable element that a user interacts with instead its being used as a way to prevent event propagation